src/test/java/com/sleekbyte/tailor/functional/ParallelTest.java

```diff
@@ -0,0 +1,108 @@
+package com.sleekbyte.tailor.functional;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertThat;
+
+import com.sleekbyte.tailor.Tailor;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * Tests for parallel analysis.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public final class ParallelTest  {
+
+    private static final String TEST_INPUT_DIR = "src/test/swift/com/sleekbyte/tailor/functional/";
+    private static final String NEWLINE_REGEX = "\\r?\\n";
+
+    private ByteArrayOutputStream outContent;
+    private List<File> inputFiles;
+    private List<String> expectedMessages;
+
+    @Before
+    public void setUp() throws ClassNotFoundException, NoSuchMethodException, IllegalAccessException,
```

Comment:
It appears that all of these exceptions may be replaced with just
```java
throws UnsupportedEncodingException
```
There is also an extra space before the opening brace.